### ISSUE TITLE
Change default site to lantasy.com not example.org

### ISSUE THREAD

**Issue by cdobbyn** [BCGamer/website#68]
Some internal links from CMS are pointing at example.org.

**Comment by cdobbyn** [comment]
This should be set via default.py

**Comment by cdobbyn** [comment]
Just set on main page itself. Preliminary research not looking good for setting this somewhere in code.

**Comment by cdobbyn** [comment]
This is for testing purpose, still needs to be done. Marking as nice to have.

**Comment by cdobbyn** [comment]
This is no longer required after #285
